app/client/requirements.txt

```diff
@@ -0,0 +1,7 @@
+autoflake
+black
+build
+grpcio==1.38.0
+isort
+protobuf
+twine
```

Comment:
What is twine and what is it used for?

Reply:
`twine` is a tool for uploading the package to pypi